scripts/test/Calibration/tofpd/test_group_calibration.py

```diff
@@ -62,32 +65,49 @@ def test_from_eng(self):
-                                                               groups,
-                                                               cc_diffcal,
-                                                               output_workspace_basename,
-                                                               PeakPositions = [1.0, 2.0, 4.0],
-                                                               PeakFunction='Gaussian',
-                                                               PeakWindow=0.4)
+        # No smoothing.
+        cc_diffcal, to_skip = group_calibration.cc_calibrate_groups(ws,
+                                                                    groups,
+                                                                    output_workspace_basename,
+                                                                    DReference=2.0,
+                                                                    Xmin=1.75,
+                                                                    Xmax=2.25)
+
+        assert_allclose(cc_diffcal.column('difc'),
+                        [starting_difc[0],
+                         starting_difc[1]/0.95,
+                         starting_difc[2]/1.05,
+                         starting_difc[4],
+                         starting_difc[5]/0.98,
+                         starting_difc[6]/1.02], rtol=0.005)
+
+        diffcal = group_calibration.pdcalibration_groups(ws,
+                                                         groups,
+                                                         cc_diffcal,
+                                                         to_skip,
+                                                         output_workspace_basename,
+                                                         PeakPositions = [1.0, 2.0, 4.0],
+                                                         PeakFunction='Gaussian',
+                                                         PeakWindow=0.4)
 
         assert_allclose(diffcal.column('difc'),
                         [starting_difc[0],
                          starting_difc[1]/0.95,
                          starting_difc[2]/1.05,
-                         0,
+                         starting_difc[3],
                          starting_difc[4]/0.95,
                          starting_difc[5]/(0.95*0.98),
                          starting_difc[6]/(0.95*1.02),
-                         0], rtol=0.005)
+                         starting_difc[7]], rtol=0.005)
 
-        assert_equal(mask.extractY(), [[0],
-                                       [0],
-                                       [0],
-                                       [1],
-                                       [0],
-                                       [0],
-                                       [0],
-                                       [1]])
+        assert_equal(mtd[f'{output_workspace_basename}_pd_diffcal_mask'].extractY(), [[0],
```

Comment:
The assertion error, when it happens, will be clearer if you use `numpy.testing.assert_equal`

Reply:
I think we are just using `numpy.testing.assert_equal` here -- if you look at the `import` statement at the top of the script, we were doing `from numpy.testing import assert_allclose, assert_equal` there.